docs/install/requirements.md

```diff
@@ -22,14 +22,15 @@ Certain versions of Harvester support the deployment of [single-node clusters](h
 
 Harvester nodes have the following hardware requirements and recommendations for installation and testing.
 
-| Type             | Requirements and Recommendations                                                                                                          |
-|:-----------------|:------------------------------------------------------------------------------------------------------------------------------------------|
-| CPU              | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above required for production                                                            |
-| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                                               |
-| Disk Capacity    | 250 GB minimum for testing (180 GB minimum when using multiple disks); 500 GB or above required for production                                                                                                                                  |
-| Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for etcd](https://www.suse.com/support/kb/doc/?id=000020100) |
-| Network Card     | 1 Gbps Ethernet minimum for testing; 10Gbps Ethernet required for production                                                                                                                       |
-| Network Switch   | Trunking of ports required for VLAN support                                                                                                                                                           |
+| Hardware | Development/Testing | Production |
+| :--- | :--- | :--- |
+| CPU | x86_64 (with hardware-assisted virtualization); 8 cores minimum | x86_64 (with hardware-assisted virtualization); 16 cores minimum |
+| Memory | 32 GB minimum | 64 GB minimum |
+| Disk capacity	| 250 GB minimum (180 GB minimum when using multiple disks) | 500 GB minimum |
+| Disk performance | 5,000+ random IOPS per disk (SSD/NVMe); management node storage must meet [etcd]((https://www.suse.com/support/kb/doc/?id=000020100)) speed requirements | 5,000+ random IOPS per disk (SSD/NVMe); management node storage must meet [etcd]((https://www.suse.com/support/kb/doc/?id=000020100)) speed requirements |
+| Network card count | Management cluster network: 2 NICs; VM workload network: 2 additional NICs (does not apply to the [witness node](../advanced/witness.md)) | Management cluster network: 2 NICs; VM workload network: 2 additional NICs (does not apply to the [witness node](../advanced/witness.md)) |
```

Comment:
We can have a single nic for testing, @mingshuoqiu any comment on this?

Reply:
The current requirement is:

>Management cluster network: 2 NICs => 

Required: Management cluster network: 1 NIC
Recommend: 2+ NIC to form a reboust with capacity of backup/loadbalance 

>VM workload network:  

Optional, depends on user network planning, there maybe 0~N such networks
For each possibile `VM workload network`, 1 indepedent NIC is a basic requirentment; 2+ NICs to form a reboust bond ...

refer: https://docs.harvesterhci.io/v1.3/networking/index#cluster-network

Reply:
Thank you. I have updated the "network card count" information.